backend/handlers/handlers.go

```diff
@@ -806,6 +809,16 @@ func (handlers *Handlers) getAccountSummary(_ *http.Request) (interface{}, error
 	return handlers.backend.ChartData()
 }
 
+// getAvailableCoinsHandler returns an array of coin codes for which you can add an account.
```

Comment:
You mean a slice, right? Also, who's "you"? :)

Reply:
Depends on your perspective, it delivers a json array :)